src/envVariables.js

```diff
@@ -204,6 +204,24 @@ let environmentVariables = {
 		optional: true,
 		default: 'rollout_manager',
 	},
+	CONSUMPTION_SERVICE_BASE_URL: {
+		message: 'Default data managers required.',
```

Comment:
@priyanka-TL - Please correct the message.

Reply:
Done